test/parallel/test-cli-eval.js

```diff
@@ -65,6 +65,18 @@ child.exec(`${nodejs} --eval "console.error(42)"`,
     assert.strictEqual(stdout, '[]\n');
     assert.strictEqual(stderr, '');
   }));
+
+  child.exec(`${cmd} "Promise.resolve().then(() => 10)"`, common.mustSucceed((stdout, stderr) => {
+    assert.strictEqual(stdout, '10\n');
+    assert.strictEqual(stderr, '');
+  }));
```

Comment:
Using `exec` is error prone, let's not if we don't have to
```suggestion
  spawnSyncAndExitWithoutError(nodejs, [s, 'Promise.resolve().then(() => 10)', {
    stdout: '10\n',
    stderr: '',
  });
```